Sources/NIOWebSocketServer/main.swift

```diff
@@ -167,7 +165,7 @@ private final class WebSocketTimeHandler: ChannelInboundHandler {
             // then, when we've sent it, close up shop. We should send back the close code the remote
             // peer sent us, unless they didn't send one at all.
             var data = frame.unmaskedData
-            let closeDataCode = data.readSlice(length: 2) ?? context.channel.allocator.buffer(capacity: 0)
+            let closeDataCode = data.readSlice(length: 2) ?? ByteBuffer()
```

Comment:
Do you want to make this change?  Feels like it encourages people away from channel.allocator even when they have one.

Reply:
If you want, I can change it but this `ByteBuffer()` thing will never allocate and we never do anything, it just needs an empty `ByteBuffer` here. But  `context.channel.allocator.buffer(capacity: 0)` also never allocates (as of this PR) so I can change it if you want.

Reply:
@PeterAdams-A Please send me an email/Slack ping if you want me to do this in a follow-up.